### DISCUSSION TITLE
Where are the compiled binaries?

### DISCUSSION THREAD

**Discussion by Milincho** [fxsound2/fxsound-app#139] [General]
Why there are only 'source code' assets but no binaries?

**Comment by raddyst** [comment]
> Why there are only 'source code' assets but no binaries?

https://github.com/fxsound2/fxsound-app/archive/refs/tags/test_release.zip

**Reply by Milincho** [reply]
> > Why there are only 'source code' assets but no binaries?
> 
> https://github.com/fxsound2/fxsound-app/archive/refs/tags/test_release.zip

![image](https://github.com/user-attachments/assets/3b1c4517-c03d-4da5-bd9f-c75a2307474d)

**Reply by raddyst** [reply]
@bvijay74 deleted last commits

**Reply by Milincho** [reply]
So, same question...     why there are no binaries available?

**Reply by raddyst** [reply]
> So, same question... why there are no binaries available?

last beta 1.1.24 - https://github.com/fxsound2/fxsound-app/raw/beta/release/fxsound_setup.exe

**Comment by bvijay74** [comment]
We have to make changes to the installer and update configuration file for moving the update URLs to GitHub. I had to make some test builds from a test branch to verify this change. You can ignore the releases with test_release tag. Sorry for any confusion caused by this test build. The beta release still remains as 1.1.24.0.
